RELEASE.md

```diff
@@ -0,0 +1,17 @@
+**Release Support Policy**
+========================
+
+## **Release Cycle**
+- We do not currently have a fixed release cycle.
+
+## **Support Duration**
+- Three releases: We currently support up to three releases at any given point in time. This means patches will be released for the most recent major release, as well as up to two prior ones.
+
+## **Documentation**
+- For each minor version (e.g., 1.6, 1.7) and the development version (corresponding to the main branch), detailed release notes will be provided.
+
+## **Compatibility**
+-	OpenTofu 1.6.2 is compatible with Terraform 1.5.x or lower. ([Migration guide](https://opentofu.org/docs/intro/migration/terraform-1.5-or-lower/))
```

Comment:
Can we remove these individual comparability notes and point instead at the migration guide?

Reply:
Hi @cam72cam , thanks for reviewing. I have made the suggested change, could you please take a look?